controllers/blackduck_mutator.go

```diff
@@ -149,24 +155,33 @@ func (p *BlackduckPatcher) patchSealKey() error {
 
 // TODO: common with Alert
 func (p *BlackduckPatcher) patchExposeService() error {
+
 	// TODO use contansts
-	id := fmt.Sprintf("Service.%s-blackduck-webserver-exposed", p.blackDuckCr.Name)
-	runtimeObject, ok := p.mapOfUniqueIdToBaseRuntimeObject[id]
+	routeID := fmt.Sprintf("Route.%s-blackduck-webserver-exposed", p.blackDuckCr.Name)
```

Comment:
please verify the route name of what we kept for 2019.8.x release

Reply:
Nice catch